perfdash/google-gcs-downloader.go

```diff
@@ -66,6 +67,19 @@ func NewGoogleGCSDownloader(opt *GoogleGCSDownloaderOptions) (*GoogleGCSDownload
 
 // TODO(random-liu): Only download and update new data each time.
 func (g *GoogleGCSDownloader) getData() (JobToCategoryData, error) {
+	configPaths := g.Options.ConfigPaths
```

Comment:
It's strange to have a get*() function modify something and this will modify the g.Options.ConfigPaths.
Moreover, if you call it twice you'll have the github config paths listed twice in the  g.Options.ConfigPaths.

Copy the slice instead?

Reply:
Good idea, thanks, done